src/sidebar/store/create-store.ts

```diff
@@ -158,14 +158,21 @@ export function createStore<
     thunk,
   ];
 
-  const enhancer = redux.applyMiddleware(...defaultMiddleware, ...middleware);
+  // In debug builds, allow using redux dev tools
+  const isDebug = process.env.NODE_ENV !== 'production';
+  const composeEnhancers =
+    (isDebug && (window as any).__REDUX_DEVTOOLS_EXTENSION_COMPOSE__) ||
```

Comment:
Do you have a documentation link for this?

Reply:
I got this from the official documentation https://github.com/reduxjs/redux-devtools/tree/main/extension#11-basic-store